romancal/datamodels/schemas/reference_files/keyword_gainfact.schema.yaml

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/keyword_gainfact.schema"
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      exposure:
```

Comment:
I see a potential conflict here - I put in a not-yet-approved PR #120 earlier this evening for this file, but my version lacks lines 8-10.

Reply:
Did you change how the object is referred to in the code? 8-10 affect the location (meta.exposure.gain_factor).

Reply:
Good point. I am revising the schema to match the code.